index.html

```diff
@@ -358,66 +316,149 @@
       </section>
       <section>
         <h2>
-          <code>CanvasImageSource</code> typedef
+          <code>DepthData</code> interface
         </h2>
         <div class="note">
           <p>
-            Several methods in the <a><code>CanvasRenderingContext2D</code></a>
-            API take the union type <a><code>CanvasImageSource</code></a> as an
-            argument. This specification extends the list of <a href=
-            "http://www.w3.org/html/wg/drafts/2dcontext/master/#image-sources-for-2d-rendering-contexts">
-            image sources for 2D rendering contexts</a> defined in
-            [[!2DCONTEXT2]].
+            Depth cameras usually produce 16-bit depth values per pixel.
+            However, neither the canvas drawing surface used to draw and
+            manipulate 2D graphics on the web platform nor the
+            <code><a>ImageData</a></code> interface used to represent image
+            data support 16 bpp.
+          </p>
+          <p>
+            To address the issue, this specification defines a new
+            <code><a>DepthData</a></code> interface.
           </p>
         </div>
-        <p>
-          A <code>video</code> element whose source is a
-          <code><a>MediaStream</a></code> object containing a <a>depth
-          track</a> is said to be a <dfn>depth video element</dfn>.
-        </p>
-        <p>
-          A <a>depth video element</a> may be used as a
-          <a><code>CanvasImageSource</code></a>.
-        </p>
+        <dl class="idl" title="interface DepthData">
+          <dt>
+            readonly attribute unsigned long width
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            readonly attribute unsigned long height
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            readonly attribute string type
+          </dt>
+          <dd>
+            -
+          </dd>
+          <dt>
+            readonly attribute string format
```

Comment:
We need to define the valid values for this attribute, say "RangeInverse" and "RangeLinear".